src/components/HealthMarkers.tsx

```diff
@@ -0,0 +1,53 @@
+// maps over 'healthMarkers' array
+// 1. checks 'type' to render icon and heading
+// 2. renders 'MarkerList' component, which will render individual cards
+
+// had to reformat data a little bit to optimize. 'healthMarkers' is an array of objects. Each object has a 'type' key that allows us to map all 3. The individual 'markers' data within each type is another array that we can esaily map through to render data.
+
+'use client'
+import healthData from '@/app/cohort_template'
+import styles from '../app/results/page.module.css'
```

Comment:
CSS Modules are a great opportunity for 2 things:
1. They break large stylesheets into smaller component specific stylesheets 
2. They create a special hash on the classnames which prevents style collisions.

To leverage the power of CSS modules we probably want to create a new `healthMarkers.module.css` file for the stylings and update this line to `import styles from '../app/results/healthMarkers.module.css`.